setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name = 'pyIATI',
-    version = '0.2.0',
+    version = '0.3.0',
```

Comment:
Anyone know it's trying to update this? Is requires.io not referencing the correct branch...

Reply:
Was a problem with requires.io having branched off quite a way back in time, so the diffs becoming a bit confused. Merging in the latest from `dev` has resolved this.